src/app/config.rs

```diff
@@ -25,11 +26,28 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Self {
-        // default preset path
+        // get paths to presets and textures
+        // TODO: get from config file or env
+        //
+        // use /usr/local/share/projectM if it exists, otherwise use local paths
+        let resource_dir = "/usr/local/share/projectM";
```

Comment:
I'd suggest not to hard-code the path, but somehow make it configurable via the build system. If installed via a package manager in Linux for example, the path would most probably be `/usr/share/projectM`, or if installed via some other means, it should be `/opt/projectM` as well.
Another way of getting the path would be by reading a config file, e.g. `/etc/projectM-SDL-RS.conf` or similar, to  retrieve the proper path.

Reply:
absolutely agree, this was always the path for earlier versions though. that's why there's the TODO. i'm not sure how to make it configurable during build but i'm sure rust has a simple way.